examples/lighting-app/lighting-common/BUILD.gn

```diff
@@ -46,6 +48,14 @@ source_set("lighting-common") {
     "gen/callback-stub.cpp",
   ]
 
+  if (chip_app_use_interaction_model) {
+    sources += [
+      "${chip_root}/src/app/clusters/im-compatibility/compatibility-command-handlers.cpp",
+      "${chip_root}/src/app/clusters/im-compatibility/ember-compatibility-functions.cpp",
+      "gen/IMClusterCommandHandler.cpp",
+    ]
+  }
```

Comment:
Can you add  a `else` case that adds `"gen/call-command-handler.cpp"`, `"${chip_root}/src/app/util/process-cluster-message.cpp"`, `"${chip_root}/src/app/util/process-global-message.cpp"` conditionally since this is basically what the current code is trying to replace. Or does it fails to link ?

Reply:
These files are already in the general source list since we cannot handle the global messages now, the two stack should be able to co-exist for a while.